server/core/connector.go

```diff
@@ -132,7 +135,21 @@ func (conn *BridgeConnector) initKafka() {
 		Timeout:   time.Duration(conn.bridge.config.ConnectTimeout) * time.Millisecond,
 		DualStack: true,
 	}
-	tlsC, err := conn.config.TLS.MakeTLSConfig()
+	var err error
+	var tlsC *tls.Config
+	if conn.config.SASL.User == "" && conn.config.SASL.Password == "" {
```

Comment:
Can empty password be legit (no comment on security)?

Reply:
Meaning should we just check SASL user?

Reply:
Fair question, don't think a blank password is legit.  Will change the check to only be for SASL userid.

Reply:
changed, commited, and squashed

Reply:
Fair call out.  Don't think it is legit.
Removed check for blank password.
commited and squashed